staging/src/k8s.io/client-go/tools/cache/controller.go

```diff
@@ -181,7 +181,7 @@ func (c *controller) LastSyncResourceVersion() string {
 	return c.reflector.LastSyncResourceVersion()
 }
 
-// processLoop drains the work queue.
+// processLoop drains the  queue.
```

Comment:
there's still an empty space here.

I'm not sure this change makes sense? it _is_ a work queue, isn't it?

Reply:
You are right, my previous idea was wrong, I will close it。